src/main/java/com/academia/library/dto/AuthorDto.java

```diff
@@ -0,0 +1,23 @@
+package com.academia.library.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@Builder
+@Setter
+@Getter
+@ToString
+@EqualsAndHashCode
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
Why do you use @AllArgsConstructor and @NoArgsConstructor ???

Reply:
Ok, thank